packages/at_secondary_server/lib/src/verb/handler/enroll_verb_handler.dart

```diff
@@ -365,8 +386,8 @@ class EnrollVerbHandler extends AbstractVerbHandler {
   /// Verifies whether the enrollment state matches the intended state.
   /// Throws AtEnrollmentException: If the enrollment state is different
   /// from the intended state.
-  void _verifyEnrollmentStateBeforeAction(
-      String? operation, EnrollStatus enrollStatus) {
+  void _verifyEnrollmentState(String? operation, EnrollStatus enrollStatus) {
+    /// ToDo: should error codes be set instead of exceptions?
```

Comment:
yes..set new error codes for each scenario

Reply:
addressed in [6cd81fa](https://github.com/atsign-foundation/at_server/pull/1562/commits/6cd81faef234f9eb77d2743856046f94e27720ee)